brain/python/default_docstrings.cpp

```diff
@@ -0,0 +1,97 @@
+// File automatically generated by /home/hernando/dev/src/config.stable/Pydoxine/CMake/../docstrings/sphinx_ext/cpp_docstrings.py
```

Comment:
That means you have to generate the docstring on some other machine before so that CI documentation upload does something useful? Does not sound right, I'd keep it simple here and update the CI if possible.